tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -50,12 +50,18 @@ var pc *pbm.Client
 var spareSpace int64 = 200
 
 var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation tests", func() {
-
 	f := framework.NewDefaultFramework("e2e-spbm-policy")
+	if supervisorCluster {
+		framework.TestContext.DeleteNamespace = false
+	}
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 	var (
-		client    clientset.Interface
-		namespace string
+		client          clientset.Interface
+		namespace       string
+		eztVsandPvcName = "pvc-vsand-ezt"
+		lztVsandPvcName = "pvc-vsand-lzt"
+		eztVsandPodName = "pod-vsand-ezt"
+		lztVsandPodName = "pod-vsand-lzt"
```

Comment:
In case of vsan direct, WCP always creates storage class with Wait For First Consumer and for the pvc to be placed in storagepool, it requires certain annotations which involves podName , hence have hardcoded the names.

  annotations:
    volume.beta.kubernetes.io/storage-class: ezt-storage-policy
    placement.beta.vmware.com/storagepool_antiAffinityRequired: pod-ez